src/BCrypt/BCrypt.cs

```diff
@@ -108,6 +108,54 @@ public static partial class BCrypt
+        /// </param>
+        /// <param name="phHash">
+        /// A pointer to a <see cref="SafeHashHandle"/> value that receives a handle that represents the multi-hash state.
+        /// This handle is used in subsequent operations such as <see cref="BCryptProcessMultiOperations(SafeHashHandle, MultiOperationType, BCRYPT_MULTI_HASH_OPERATION[], int, int)"/>.
+        /// When you have finished using this handle, release it by passing it to the <see cref="BCryptDestroyHash(IntPtr)"/> function.
+        /// </param>
+        /// <param name="nHashes">
+        /// The number of elements in the array. The multi-hash state that this function creates is able to perform parallel computations on nHashes different hash states.
+        /// </param>
+        /// <param name="pbHashObject">
+        /// A pointer to a buffer that receives the multi-hash state.
+        /// If <paramref name="pbHashObject"/> is NULL and <paramref name="cbHashObject"/> has a value of zero (0), the object buffer is automatically allocated.
+        /// </param>
+        /// <param name="cbHashObject">
+        /// The size of the <paramref name="pbHashObject"/> buffer, or zero if <paramref name="pbHashObject"/> is NULL.
+        /// </param>
+        /// <param name="pbSecret">
+        /// A pointer to a buffer that contains the key to use for the hash or MAC.
+        /// The <paramref name="cbSecret"/> parameter contains the size of this buffer.
+        /// This key only applies to hash algorithms opened by the <see cref="BCryptOpenAlgorithmProvider(out SafeAlgorithmHandle, string, string, BCryptOpenAlgorithmProviderFlags)"/> function by using the <see cref="BCryptOpenAlgorithmProviderFlags.BCRYPT_ALG_HANDLE_HMAC_FLAG"/> flag
+        /// Otherwise, set this parameter to NULL.
+        /// The same key is used for all elements of the array.
+        /// </param>
+        /// <param name="cbSecret">
+        /// The size, in bytes, of the <paramref name="pbSecret"/> buffer. If no key is used, set this parameter to zero.
+        /// </param>
+        /// <param name="dwFlags">
+        /// Flags that modify the behavior of the function. This can be zero or the values below.
+        /// Multi-hash objects are always reusable and always behave as if the <see cref="BCryptCreateHashFlags.BCRYPT_HASH_REUSABLE_FLAG"/> was passed. This flag is supported here for consistency.
+        /// </param>
+        /// <returns>Returns a status code that indicates the success or failure of the function.</returns>
+        [DllImport(nameof(BCrypt), SetLastError = true)]
+        public static extern NTSTATUS BCryptCreateMultiHash(
+            SafeAlgorithmHandle hAlgorithm,
+            out SafeHashHandle phHash,
+            int nHashes,
+            byte[] pbHashObject,
```

Comment:
I believe you should use these friendly flags and a native pointer to give more method overloads and flexibility to the caller:
```suggestion
            [Friendly(FriendlyFlags.Out | FriendlyFlags.Optional)] byte* pbHashObject,
```

Reply:
I've taken care of this.